### PR TITLE
changed the sceduling to the sceduling package

### PR DESCRIPTION
**Pull request by CommanderStorm**
Reason for this Pull request:

- no more birthdays at 16:xx
- no more mensa updates at 16:xx (dont know if that happened, but that could have happened)
- sceduling the python way

Drawbacks:

- need to reload the module
- posibility less output of errors (no functionnames) on the console, 
as only the errormessage can be outputet

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -26,26 +27,19 @@ async def on_ready(self):
...
-                except Exception as e:
-                    print(f"Exception while running job {job['function'].__name__}:")
-                    print(f"{type(e).__name__}: {e}")
+            try:
+                schedule.run_pending()
+            except Exception as e:
+                print(f"{type(e).__name__}: {e}")
             time.sleep(10)
 
+    def register_job_dayly(self, daytime, f):
```
Comment (timschumi) [inline bot.py]: daily

**Review by timschumi — CHANGES_REQUESTED** [review]

**Comment by timschumi** [comment]
Also, could we do this by assigning a new Scheduler to bot explicitly instead of using the default Scheduler instance?

**Comment by CommanderStorm** [comment]
tested this. 
It (status+birthday) works.

(mensa does not really work, but that is the api's/virus fault for disrupting the whole eating on campus thing..)

**Review by timschumi — APPROVED** [review]
Looks fine.

**Pull request merged**